libs/blocks/carousel/carousel.js

```diff
@@ -179,6 +180,47 @@ function setIndicatorMultiplyer(carouselElements, activeSlideIndicator, event) {
   }
 }
 
+function updateAriaLive(ariaLive, slide) {
+  let text = '';
+  slide.querySelectorAll(':scope > :not(.section-metadata').forEach((el, index) => {
+    text += `${index ? ' ' : ''}${el.textContent.trim()}`;
+  });
+  if (text) {
+    ariaLive.textContent = text;
+  } else {
+    const img = slide.querySelector('img[alt]');
+    const video = slide.querySelectorAll('video[title], iframe[title]');
+    ariaLive.textContent = img?.getAttribute('alt') || video?.getAttribute('title') || '';
```

Comment:
Since you have `querySelectorAll` here, you can't call `video?.getAttribute('title')` on a nodeList... or am I missing something?

So IMO you wanted to do
```suggestion
    const el = slide.querySelector('img[alt], video[title], iframe[title]');
    ariaLive.textContent = el?.getAttribute('alt') || el?.getAttribute('title') || '';
```

Reply:
great catch, thank you !